src/intrinsics/react-mocks/mocks.js

```diff
@@ -29,34 +75,126 @@ let reactCode = `
         return true;
       }
       getChildContext() {}
-    },
-    createElement: function() {
-      // TODO
-    },
-    cloneElement(element, config, children) {
+    }
+
+    PureComponent.prototype.isPureReactComponent = true;
```

Comment:
Shouldn't it also have `isReactComponent`?

Reply:
Added. :)